cpp/open3d/visualization/rendering/Scene.h

```diff
@@ -41,83 +42,110 @@ class AxisAlignedBoundingBox;
 namespace visualization {
 namespace rendering {
 
+class Renderer;
 class View;
+class Model;
+class Material;
```

Comment:
Material seems to hold most of the properties for visualization. 
Would it make sense to wrap it in a more generic class ```Properties```, which can store more properties like ```visible, rendermode(wire,solid), use_lod, material, ...```? We could also have ```Properties``` as a base class and subclass for Geometry, Lights, and Cameras.
With this we can then write imgui code to let the user change all of these properties.